lib/install/config/loaders/installers/elm.js

```diff
@@ -1,5 +1,9 @@
+const path = require('path')
+
+const elmSource = path.join(__dirname, '..', '..', '..')
```

Comment:
Is this elm app source directory? We can just use `path.resolve(paths.source)` from `paths.yml`